### ISSUE TITLE
Google maps not working in c:geo

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#15932]
### Describe your problem!

Starting today I experience problems to use Google maps in c:geo.
It behaves somehow strange and not reproducible:
- Google:Map only shows a very very basic maps (some streets, no details) or when trying several times it shows nothing at all
- Google:Sat shows a bit more detailed map (however no satellite pictures at all)
- Google:Terrain shows the same as Google:Sat

Happens on both my devices using current nightlies.
Powering up the emulator right now to take a closer look.

Does Google have a problem or did we change something here recently?

### How to reproduce?

unclear

### Reproducible

Unclear

### c:geo Version

2024.07.29-NB

**Comment by Lineflyer** [comment]
What I see in the log of my tablet device:

```
2024-07-29 21:37:56.928 19606-22307 Proxy...kend cgeo.geocaching  W  Too many Flogger logs received before configuration. Dropping old logs.
2024-07-29 21:37:56.928 19606-22307 Proxy...kend cgeo.geocaching  W  Too many Flogger logs received before configuration. Dropping old logs.
2024-07-29 21:37:56.928 19606-22307 Proxy...kend cgeo.geocaching  W  Too many Flogger logs received before configuration. Dropping old logs.
2024-07-29 21:37:56.964 19606-19606 Proxy...kend cgeo.geocaching  W  Too many Flogger logs received before configuration. Dropping old logs.
2024-07-29 21:37:56.964 19606-19606 Proxy...kend cgeo.geocaching  W  Too many Flogger logs received before configuration. Dropping old logs.
2024-07-29 21:37:57.145 19606-22320 dej          cgeo.geocaching  W  Couldn't create marker: _google_maps_sdk_developer_exception_
2024-07-29 21:37:57.395 19606-22320 dej          cgeo.geocaching  W  Couldn't create marker: _google_maps_sdk_developer_exception_
2024-07-29 21:37:57.645 19606-22320 dej          cgeo.geocaching  W  Couldn't create marker: _google_maps_sdk_developer_exception_
2024-07-29 21:37:57.895 19606-22320 dej          cgeo.geocaching  W  Couldn't create marker: _google_maps_sdk_developer_exception_
```

**Comment by moving-bits** [comment]
Just cross-checked with 2024-07-26-RC: no problems with either of the GM types (Android 14 device)

**Comment by Lineflyer** [comment]
Strange...I can't get it working. No idea why.

GMap app:
![image](https://github.com/user-attachments/assets/1797bbb6-3883-4fd7-9a7e-c7f57f1d4700)


c:geo:
![image](https://github.com/user-attachments/assets/eddde65b-14d0-4320-bebe-093a41882c90)

**Comment by moving-bits** [comment]
![Screenshot_20240729_215125_cgeo](https://github.com/user-attachments/assets/a2f03be0-a1c6-4702-b06c-69d077dffe1a)

**Comment by Lineflyer** [comment]
Found the problem, but can't explain the problem:

I noticed that Google maps are working within c:geo once I change from my home Wifi to mobile network.
Interesting enough the GMaps app was working in both networks without any issue.

Rebooting the router solved the issue in c:geo.
Now I can no longer reproduce, thus I can't dive deeper into this.

What could be the difference between GMaps in c:geo and in the native app?
Could it be the protocol stack (IPv4 vs IPv6)? Because my router has some IPv6 issues from time to time.

Anyway closing for now...but if someone has any idea, feel free to comment.
